packages/react-core/src/components/Select/Select.tsx

```diff
@@ -131,6 +137,21 @@ const SelectBase: React.FunctionComponent<SelectProps & OUIAProps> = ({
   }, [isOpen]);
 
   React.useEffect(() => {
+    const onToggleArrowKeydownDefault = (event: KeyboardEvent) => {
+      event.preventDefault();
+
+      let listItem: HTMLLIElement;
+      if (event.key === 'ArrowDown') {
+        listItem = menuRef.current?.querySelector('li');
+      } else {
+        const allItems = menuRef.current?.querySelectorAll('li');
+        listItem = allItems ? allItems[allItems.length - 1] : null;
+      }
+
+      const focusableElement = listItem?.querySelector('button:not(:disabled),input:not(:disabled)');
```

Comment:
This query selector should also include `a:not([aria-disabled="true"])` similar to `Dropdown`/`MenuContainer` as `SelectOptions` can also be links. If you remove the first two options from the "Select option variants" example so the link option is first pressing arrow down does nothing.